app/views/jobs/index.html.erb

```diff
@@ -1,31 +1,28 @@
-<h1> Jobs!</h1>
+<% if user_signed_in? %>
 
-<header style="text-align: right">
-  <%= button_to "New Job", new_job_path, method: :get %>
-</header>
+  <h2 class="header">
+    <div style="text-align: center"> Jobs!</div>
 
-<h2> Interested</h2>
-  <% @jobs.select { |job| job.has_status?("Interested") }.each do |job| %>
-    <%= render partial: "job_card_partial", locals: { job: job } %>
-<% end %>
+    <div style="text-align: right">
+      <%= button_to "New Job", new_job_path, method: :get %>
+    </div>
+  </h2>
 
-<h2> Applied</h2>
-  <% @jobs.select { |job| job.has_status?("Applied") }.each do |job| %>
-    <%= render partial: "job_card_partial", locals: { job: job } %>
-<% end %>
+  <% %w[Interested Applied Interviewed Offered Rejected].each do |status| %>
 
-<h2> Interviewed</h2>
-  <% @jobs.select { |job| job.has_status?("Interviewed") }.each do |job| %>
-   <%= render partial: "job_card_partial", locals: { job: job } %>
-<% end %>
+    <% if @jobs.any? { |job| job.status == status } %>
+      <h3> <%= status  %></h3>
+      <% @jobs.select { |job| job.has_status?(status) }.each do |job| %>
+        <%= render partial: "job_card_partial", locals: { job: job } %>
```

Comment:
It's weird that the name is `job_card_partial`.

I think it should just be `job` - you know it's a partial because the file starts with an underscore so you don't need to explicitly name it that 

I think it might have been a previous PR that made that refactor but just calling it out now that I noticed it

Reply:
I named it job_card_partial because i already had a job partial for the index. I might name it job_card rather than job, unless you can think of a better name?